src/libraries/System.Text.Json/tests/System.Text.Json.Tests/JsonNode/JsonArrayTests.cs

```diff
@@ -196,7 +196,7 @@ public static void CopyTo()
             Assert.Throws<ArgumentOutOfRangeException>(() => jArray.CopyTo(arr, -1));
         }
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsReflectionEmitSupported))]
         public static void ConvertJSONArrayToIListOfJsonNode()
         {
             dynamic obj = JsonSerializer.Deserialize<JsonArray>("[42]");
```

Comment:
This is a test bug. It didn't mean to test dynamic. Please just make the variable of type `JsonArray` instead. Thanks

Reply:
thanks, done